Randolph-dev/Druid_Partnering_Project_React24k
# SCRUM-157 Feat: Refactored the bloglisting code and added blogpage wi…

### In This PR;

I have added the content for the Blogs page, which was previously missing. The Blogs listing page remains unchanged but the code has been refactored to support the addition of individual blog content pages.

### Changes Made;

- Added content to individual blog pages.
- Refactored the code for the Blogs listing page to enable navigation to individual blog content pages.
- Added a fronted types file in the types folder

### Testing

- Import the latest db with the blogs content (1.5.gz)
- lando drush cr
- Check out this branch.


### Visit the Blogs page and check whether;

- The Blogs listing page functions as expected.
- Each blog item links to its respective content page.
- The content on individual blog pages is displayed correctly.
